src/Instance/InstanceEdit/InstanceEdit.test.js

```diff
@@ -56,7 +63,6 @@ const InstanceEditSetup = () => (
             mutator={{
```

Comment:
You removed the mutator from the component's props, do you need it in your test still?